inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/util/QueryUtil.java

```diff
@@ -71,6 +71,7 @@ public static TupleQuery generateCandidateQuery(RepositoryConnection conn, List<
             "      {",
             "        ?e2 ?labelpredicate ?altLabel.",
             "        ?altLabel bif:contains '?entityLabel'. ",
+            "        ?e2 schema:description ?description.",
```

Comment:
Ok for now. Please open an issue to make the description IRI configurable in the KB settings.